src/codegen/generate-classes.ts

```diff
@@ -1810,10 +1807,7 @@ const JavaScriptCoreBindings = struct {
           output += `
         pub fn ${names.fn}(thisValue: *${typeName}, globalObject: *JSC.JSGlobalObject, callFrame: *JSC.CallFrame${proto[name].passThis ? ", js_this_value: JSC.JSValue" : ""}) callconv(JSC.conv) JSC.JSValue {
           if (comptime Environment.enable_logs) zig("<d>${typeName}.<r>${name}<d>({})<r>", .{callFrame});
-          return @call(.always_inline, ${typeName}.${fn}, .{thisValue, globalObject, callFrame${proto[name].passThis ? ", js_this_value" : ""}}) catch |err| switch (err) {
-            error.JSError => .zero,
-            error.OutOfMemory => globalObject.throwOutOfMemoryValue(),
-          };
+          return JSC.toJSHostValue(globalObject, ${typeName}.${fn}(thisValue, globalObject, callFrame${proto[name].passThis ? ", js_this_value" : ""}));
```

Comment:
The inline was intentional. We cannot add extra functions to the call stack. Please revert this